src/components/game/GameCard.vue

```diff
@@ -66,7 +63,7 @@ export default {
       unobserve(el.value)
     })
 
-    return { game, platforms, el, isShown }
```

Comment:
We don't need to return the _game_ object since it is already received as a prop 😉